src/Avalonia.Controls/Window.cs

```diff
@@ -216,22 +217,25 @@ public void Close(object dialogResult)
         /// <summary>
         /// Hides the window but does not close it.
         /// </summary>
-        public void Hide()
+        public override void Hide()
         {
             using (BeginAutoSizing())
             {
                 PlatformImpl.Hide();
```

Comment:
Shouldn't Show/Hide calls be inside IsVisible change handler? So people could bind that property to a view model.